src/diffusers/loaders/unet.py

```diff
@@ -910,10 +916,11 @@ class FromOriginalUNetMixin:
 
     @classmethod
     @validate_hf_hub_args
-    def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
+    def from_single_file(cls, pretrained_model_link_or_path: Optional[str] = None, **kwargs):
```

Comment:
I don't understand here 
why do we need both  `FromOriginalModelMixin` and` UNet2DConditionLoadersMixin` has a `from_single_file` method?

Reply:
Still need to remove this class `FromOriginalUNetMixin` 😅